gp2gp-translator/src/integrationTest/java/uk/nhs/adaptors/pss/translator/service/BundleMapperServiceIT.java

```diff
@@ -195,4 +212,12 @@ private List<DocumentReference> extractDocumentReferencesFromBundle(Bundle bundl
             .map(DocumentReference.class::cast)
             .toList();
     }
+
+    private List<Observation> extractObservationsFromBundle(Bundle bundle) {
+        return bundle.getEntry().stream()
+            .map(Bundle.BundleEntryComponent::getResource)
+            .filter(resource -> resource.getResourceType().equals(ResourceType.Observation))
+            .map(Observation.class::cast)
+            .toList();
+    }
```

Comment:
I can see this being a useful chunk of functionality which may be used elsewhere within the tests, would it be worth extracting this out to another class so that we can decouple it from `BundleMapperServiceIT.java`?